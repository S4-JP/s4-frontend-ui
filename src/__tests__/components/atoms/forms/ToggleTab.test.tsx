import React from 'react'
import { render } from '@testing-library/react'
import { ToggleTab } from '../../../../index'

/**
 * type Props = {
 *   label?: string
 *   checked?: boolean
 *   onChange?: (e: React.SyntheticEvent) => void
 * }
 */
describe('form - ToggleTab のレンダリング確認', () => {
  test('オプションなしで設定した場合、初期値でレンダリングされる。', () => {
    const { getByTestId } = render(<ToggleTab />)

    expect(getByTestId('atoms-f-toggle-tab-label')).toMatchSnapshot()
  })
})
