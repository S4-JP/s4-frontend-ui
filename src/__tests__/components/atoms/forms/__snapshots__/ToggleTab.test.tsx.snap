// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form - ToggleTab のレンダリング確認 オプションなしで設定した場合、初期値でレンダリングされる。 1`] = `
<label
  class="sc-efBctP eqbWMO"
  data-testid="atoms-f-toggle-tab-label"
>
  <input
    data-testid="atoms-f-toggle-tab-input"
    type="radio"
  />
  <span
    data-testid="atoms-f-toggle-tab-span"
  />
</label>
`;
